apps/prairielearn/src/pages/news_items/news_items.html.ts

```diff
@@ -0,0 +1,87 @@
+import { z } from 'zod';
+
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { NewsItemSchema } from '../../lib/db-types.js';
+
+export const NewsItemRowSchema = NewsItemSchema.extend({
+  formatted_date: z.string(),
+  show_student_badge: z.boolean(),
+  unread: z.boolean(),
+});
+type NewsItemRow = z.infer<typeof NewsItemRowSchema>;
+
+export function NewsItems({
+  resLocals,
+  newsItems,
+}: {
+  resLocals: Record<string, any>;
+  newsItems: NewsItemRow[];
+}) {
+  const { urlPrefix, news_item_notification_count: newsItemNotificationCount } = resLocals as {
+    urlPrefix: string;
+    news_item_notification_count?: number | null;
+  };
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../partials/head') %>", {
+          ...resLocals,
+          pageTitle: 'News',
+          pageNote:
+            newsItemNotificationCount ?? 0 > 0 ? `${newsItemNotificationCount} Unread` : undefined,
```

Comment:
The original did not specify the page title, which means the title would usually defer to the course/instance name instead. It did specify the page note, though, but the page note only applies if the page title is provided, so I now added a proper page title.